### ISSUE TITLE
Custom screen resolution issues when changing back to native

### ISSUE THREAD

**Issue by fr500** [libretro/RetroArch#8441]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

If you have a custom fullscreen resolution set (in my case, 1920x1080 whilst native is 3840x2160), and you decide you now want to use native again, then you exit RetroArch, RA will set your desktop to the improper 1920x1080.

### Expected behavior

Desktop resolution after quitting RetroArch should be untouched

### Actual behavior

[What is actually happening]

### Steps to reproduce the bug

1. Start RetroArch
2. Change resolution to a custom resolution
3. Exit
4. Start again
5. Change resolution to your original desktop resoluton
6. Exit
7. (╯°□°）╯︵ ┻━┻

### Bisect Results

[Try to bisect and tell us when this started happening]

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.7.6/234423bc55

### Environment information

- OS: Windows 10
- Compiler: [In case you are running local builds]

**Comment by orbea** [comment]
Is this only windows? I haven't noticed it with xorg.

**Comment by fr500** [comment]
Right, I don't think resolution switchin like this:
Is hooked up on linux 

![image](https://user-images.githubusercontent.com/1721040/54094206-2901f080-436d-11e9-8cee-f4f451928d37.png)

I may be wrong tho.

**Comment by orbea** [comment]
Yes, you're right. It only shows `0x0` here now that I looked again.
